src/AP.AzureADAuth/ViewModels/LoginPageViewModel.cs

```diff
@@ -51,8 +51,8 @@ public void Destroy()
         {
             _isBusyHelper.Dispose();
             _isBusyHelper = null;
-            LoginCommand.Dispose();
-            LoginCommand = null;
+            //LoginCommand.Dispose();
+            //LoginCommand = null;
```

Comment:
I generally do not like including commented out code in PR's unless there is a specific reason we want it... that said why are we commenting this out?

Reply:
I have removed this commented lines in my next commit.